expression/expression.go

```diff
@@ -1156,6 +1156,11 @@ func scalarExprSupportedByFlash(function *ScalarFunction) bool {
 		}
 	case ast.IsTruthWithNull, ast.IsTruthWithoutNull, ast.IsFalsity:
 		return true
+	case ast.GetFormat:
+		switch function.Function.PbCode() {
```

Comment:
We only check PB code for fine-grained pushdown, i.e., when a function has multiple overloads and only some of the overloads should be pushed down. Since `get_format` has only one overload, checking AST only should be sufficient already.